doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -2,10 +2,10 @@
 
 ACRN Configuration Tool
 #######################
-
-.. note:: This document is under development and planned for the 1.3
-   release.
-
+ACRN configuration tool is designed for System Integrators / Tier1’s to customize
+ACRN to meet their own needs. It consists of two tools. The ``Kconfig`` tool and the 
+``acrn-config`` tool. The latter allows users to provision VMs via a WebUI and configure 
```

Comment:
I have mixed feelings about the use of "provision" here, would be "define" be a good alternative? @deb-intel